voluncheer/opportunityboard/views/search.py

```diff
@@ -69,26 +69,6 @@ def search(self):
         return filtered_opportunity
 
 
-def filter_search(request):
```

Comment:
Why are we removing filter search?

Reply:
For pagination, we can merge the search func and the display. Since other cases there's no filter submit. Reading filter won't return anything.